examples/second_set_uvs.js

```diff
@@ -0,0 +1,259 @@
+
+        const textContent = `Three Mesh-Ui`.toUpperCase();
+        const text = new ThreeMeshUI.Text({
+            content: textContent,
+        });
+        container.add(
+            text
+        );
+
+
+        // @Notes : About this example itself,
+        // I found it difficult to get started, due to nested scopes of this location.
+        // The demo/sample will really start here but the current code breadbrumd is :
+        //     makeTextPanel() > callback loader.load() > onAfterUpdate()
+        text.onAfterUpdate = function () {
+            if (text.children.length > 0) {
+
+                // 1. BOX REFERENCE
+                // the idea behind this sample is to provide a second set of uvs for each glyphs
+                // we know we will need to set each UVs and for that we need a reference.
+                // that reference can be acquired using the boundingBox of the inlineComponent
+
+                // be sure that box is computed before aquiring it
+                // @NOTES : `text.children[0]` can currently works because `mergedGeometries`
+                text.children[0].geometry.computeBoundingBox();
+                // this will return a THREE.Box3
+                const bBox = text.children[0].geometry.boundingBox;
+
+                // 2. BOX DIMENSIONS
+                // Using the reference THREE.Box3, we can compute
+                // the width and the height of the whole geometry
+                //    optionally the depth, but we don't need it for that sample
+                const geometryW = bBox.max.x - bBox.min.x;
+                const geometryH = bBox.max.y - bBox.min.y;
+                // const depth = bBox.max.z - bBox.min.z;
+
+                // 3. SECOND UV SET
+                // Base on the current vertex position and using
+                // geometryW and geometryH previously computed
+                const secondUVSetArray = [];
+
+                // loop through each vertices
+                const textVerticesArray = text.children[0].geometry.getAttribute('position').array;
+                for (let i = 0; i < textVerticesArray.length; i += 3) {
+
+                    const vertX = textVerticesArray[i];
+                    const vertY = textVerticesArray[i + 1];
+
+                    //normalized UV
+                    secondUVSetArray.push((vertX / geometryW) + 0.5);
+                    secondUVSetArray.push((vertY / geometryH) + 0.5);
+                }
+
+                // add a second set of UVs as geometry attributes
+                // Gemeotry Attributes are automatically passed to vertexShader
+                text.children[0].geometry.setAttribute('uvC', new BufferAttribute(new Float32Array(secondUVSetArray), 2));
+
+
+                // 4. Replace the current fontMaterial shader with a customized one
+                text.fontMaterial.uniforms.u_map = { value: textureC };
+                text.fontMaterial.uniforms.u_uvCTransform = {value:textureC.matrix};
+
```

Comment:
I didn't really thought a lot about how to reuse existing shaders. Instead they are currently copy pasted from three-mesh-ui MaterialManager with small additions